utils/ort/src/main/kotlin/JavaBootstrapper.kt

```diff
@@ -105,7 +105,7 @@ object JavaBootstrapper {
             Match.ANY
         )
 
-        val pkg = pkgs.sortedBy { it.id }.find { it.distributionName == distributionName }
+        val pkg = pkgs.sortedBy { it.id }.find { it.distribution != null && it.distributionName == distributionName }
```

Comment:
As `distributionName` / `getDistributionName()` just calls `distribution.getName()`, this can be combined to `find { it.distribution?.name == distributionName }`.